ui-participant/src/participant/ParticipantProfile.test.tsx

```diff
@@ -25,6 +26,25 @@ const jsalkProfile: Profile = {
   sexAtBirth: 'M'
 }
 
+const labels = {
+  off: 'Off',
+  on: 'On',
+  notProvided: 'Not provided',
+  editName: 'Edit Name',
+  editBirthday: 'Edit Birthday',
+  editDoNotSolicit: 'Edit Do Not Solicit',
+  editNotifications: 'Edit Notifications',
+  editPrimaryAddress: 'Edit Primary Address',
+  editContactEmail: 'Edit Contact Email',
+  editPhoneNumber: 'Edit Phone Number',
+  editMailingAddress: 'Edit Mailing Address',
+  editCommunicationPreferences: 'Edit Communication Preferences',
+  givenName: 'Given Name',
+  familyName: 'Family Name',
+  save: 'Save',
+  cancel: 'Cancel'
+}
```

Comment:
this seems like it's adding a layer of indirection we don't need -- the i18n should default to rendering the message key if the translation doesn't exist.  So the test assertions should be looking for the keys (which actually makes the tests more connected to the code under test)